.buildkite/scripts/dra_release.sh

```diff
@@ -41,6 +41,10 @@ if [[ ${TYPE} == "snapshot" ]]; then
     VERSION="${VERSION}-SNAPSHOT"
 fi
 
+if [[ ${FIPS} == "true" ]]; then
+    export FIPS=true
```

Comment:
What does this do ?

Reply:
Not sure why this made it it; i'll remove it